src/main.rs

```diff
@@ -15,15 +15,20 @@
 // Written by: Bobbbay Bobbayan
 
 use std::{thread, time, fs, env};
-
+use std::time::{SystemTime, UNIX_EPOCH};
 fn main() {
     let args: Vec<String> = env::args().collect();
 
+    let start = SystemTime::now();
+    let since_the_epoch = start
```

Comment:
Seems like an overdo of words, try `since_epoch` or maybe even `since_last_epoch`.

Reply:
Resolved this issue